pkg/monitor/dynkeepalived.go

```diff
@@ -42,7 +42,7 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 			if err != nil {
 				return err
 			}
-			if prevConfig == nil || prevConfig.VRRPInterface != newConfig.VRRPInterface {
+			if prevConfig == nil || prevConfig.VRRPInterface != newConfig.VRRPInterface || len(prevConfig.LBConfig.Backends) != len(newConfig.LBConfig.Backends) || prevConfig.BootstrapIP != newConfig.BootstrapIP {
```

Comment:
Do we want to update keepalived.conf after a single poll or we would like to add stabilization here?
What will happen If the DNS ( we retrieve backends from DNS server) won't reply for X seconds?

Reply:
We could put in a counter that if that section has changed for 3 polls, then go ahead and commit the change.

Reply:
We could put in a counter that if that section has changed for 3 polls, then go ahead and commit the change.

Reply:
Sounds good, we did something similar in haproxy/monitor